{{ cookiecutter.project_slug }}/api/pyproject.toml

```diff
@@ -0,0 +1,100 @@
+[tool.pytest.ini_options]
+DJANGO_SETTINGS_MODULE = "{{ cookiecutter.project_slug }}.settings"
+python_files = ["tests.py", "test_*.py", "*_test"]
+
+[tool.black]
+line-length = 120
+target-version = ['py310']
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.env
+  | \.venv
+  | _build
+  | buck-out
+  | build
+  | dist
+  | node_modules
+)/
+'''
+
+[tool.bandit]
+skips = ['B101',]
+
+[tool.isort]
+profile = 'black'
+line_length = 120
+multi_line_output = 3
+lines_between_types = 1
+lines_after_imports = 2
```

Comment:
it is not the same as it was in `.isort.cfg`, looks like we lost nice groups defined in the deleted config.

Reply:
I did it on purpose. To keep our configuration simple... You think we need rest and Django group?

Reply:
We can discuss the necessity of Django & DRF separation. But I guess we need a separation between these groups for sure:
- Python built-ins
- Django
- 3rd party
- 1st party